### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by samgnaniah**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,33 @@
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/micro-integrator)](https://wso2.org/jenkins/job/products/job/micro-integrator/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# WSO2 Micro Integrator
+
+## Introduction
+
+In recent times we have witnessed the growth of approaches like Continuous Integration and Continuous Deployment (CI/CD), which has subsequently led to the rise of microservices. Microservices are designed to start up very quickly, perform a specific task, and then shut down.
```
Comment (a5anka) [inline README.md]: This is good. But maybe we should mention that the MSA became more popular due to the popularity of cloud-native technologies and the assistance it provides for agile development.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,33 @@
...
+
+WSO2 Enterprise Integration (EI) is an open source integration platform and the Micro Integrator is a cloud native variant of WSO2 EI.
+
+The Micro Integrator is a lightweight integration framework built for developers who would like to create and run their integrations and integrate microservices. Micro Integrator provides developers the freedom to quickly develop and run their integrations in a Microservices Architecture (MSA) environment and to deploy them using CICD. So with the Micro Integrator, developers are able to develop and manage composite microservices. 
+
+Furthermore, since the startup time is a critical aspect when working with microservices, the Micro Integrator profile is designed with an improved startup time than the conventional WSO2 ESB or the ESB profile of WSO2 EI. Thus, with this improved startup time, the Micro Integrator profile becomes container-friendly for you to work with your microservices in a container-based architecture. The Micro Integrator works natively on the Kubernetes ecosystem and it makes deployment with Docker a simple process.
```
Comment (a5anka) [inline README.md]: Are we focusing too much on the startup time here? Startup time is just one aspect.

Comment (samgnaniah) [reply README.md]: Is there anything we are missing that we can add to this?

File: README.md [hunk 3]
```diff
@@ -3,6 +3,33 @@
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/micro-integrator)](https://wso2.org/jenkins/job/products/job/micro-integrator/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# WSO2 Micro Integrator
+
+## Introduction
+
+In recent times we have witnessed the growth of approaches like Continuous Integration and Continuous Deployment (CI/CD), which has subsequently led to the rise of microservices. Microservices Architecture (MSA) became more popular due to the rise of cloud native technologies and the assistance it provides for agile development. Microservices are designed to start up very quickly, perform a specific task, and then shut down.
+
```
Comment (a5anka) [inline README.md]: "Microservices are designed to start up very quickly, perform a specific task, and then shut down." suites more for serverless technologies which is a subset of the microservices. Therefore we might not be very accurate here.

File: README.md (hunk 2 above)
Comment (a5anka) [reply README.md]: My feeling is that we are giving too much focus to startup time. IMO startup time might not be the main reason why someone wants to use micro integrator. Maybe we can reorganize to reduce the focus on startup time.

**Comment by kasun04** [comment]
The content too verbose and some key areas related to micro integrator are missing. We need to rewrite this so that it concisely elaborate: 

- What is WSO2 Micro-Integrator 
- Why and when to use it? 
- Getting started guide

**Comment by samgnaniah** [comment]
Closing this. Will send a fresh PR with completely new content.

**Pull request closed without merging**